addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/api.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+    <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+    <TITLE>
+        Fuzzing API
+    </TITLE>
+</HEAD>
+<H1>Fuzzing API</H1>
+<p>
+    The API has been designed to be used by people with a range of security experience, for
+    example “simpleHttpFuzzer” is a basic fuzzer fuzzes one “Fuzz location”, extracting payloads
+    from one file, and returns the fuzzerId.
+</p>
+<h3>Homepage</h3>
+<p><a href="https://github.com/zaproxy">https://github.com/zaproxy</a><br>
+    Documentation can be found at:<br>
+    <a href="https://github.com/zaproxy/zap-api-docs/fuzz">https://github.com/zaproxy/zap-api-docs/fuzz</a>
+    <br>
+</p>
+<h3>Message Processors</h3>
+<p>
+    The default processor enabled by default for all requests is “Request Content-Length Updater”
+    which updates the content-length field according to the attack payload. Option to add more
+    processors will be added in future releases.
+</p>
+<h3>Fuzzer Options</h3>
+<p>
+    Currently these fuzzer options can be changed through the FuzzAPI but in future releases
+    more options will be added.
+</p>
+<ul>
+    <li>Retries on IO error</li>
+    <li>Max. error allowed</li>
+    <li>Payload Replacement Strategy</li>
+    <li>Concurrent scanning threads per scan</li>
+    <li>Delay when fuzzing (in milliseconds)</li>
+</ul>
+Note: options changed after the fuzzer has been initialised will remain same for that fuzzer.
+<h3>Simple Http Fuzzer</h3>
+<p>The simpleHttpFuzzer fuzzes one fuzz location, with inputs</p>
+<table style="width:100%" border="1px solid black" border-collapse="collapse">
```

Comment:
Isn't 100% default?

Reply:
It is not 100% default, but I still removed it, as it is not required.